src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            int buy = 0;
+            int supply = 0;
+            String line;
+            String[] data;
+            while ((line = reader.readLine()) != null) {
+                data = line.split(",");
+                switch (data[0]) {
+                    case "buy":
+                        buy += Integer.parseInt(data[1]);
+                        break;
+                    case "supply":
+                        supply += Integer.parseInt(data[1]);
+                }
+                try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
```

Comment:
The BufferedWriter is placed inside the while loop, which means the file will be opened and written to for each line in the input file. This will result in only the last line's data being saved in the output file. Consider moving the BufferedWriter outside the loop to write the accumulated results after processing all lines.